lib/ProductOpener/CRM.pm

```diff
@@ -81,7 +84,7 @@ my $crm_data;
 my @required_tag_labels = qw(onboarding);
 # Category (res.partner.category) must be defined in Odoo :
```

Comment:
The meaning of "category" is not very intuitive, could you rename the field to "data_sources" instead?